test/destructor/index.js

```diff
@@ -109,9 +109,9 @@ test('Destructors fire in MediaStreamTrack use-case', async t => {
 
     const [pc1, pc2] = await negotiateRTCPeerConnections({
       withPc1(pc1) {
-        stream1.getTracks().forEach(track => pc1.addTrack(track, stream1));
+        stream1.getTracks().forEach(track => pc1.addTrack(track, [stream1]));
```

Comment:
Per comment above, using rest parameter syntax, there should be no change necessary here.